docs/mediapipe.md

```diff
@@ -0,0 +1,141 @@
+# Mediapipe {#ovms_docs_mediapipe}
+
+## Introduction
+MediaPipe is a production ready framework for building pipelines to perform inference over arbitrary sensory data. Using MediaPipe in the OVSM enables user to define a powerful graph from a lot of ready calculators/nodes that come with the MediaPipe which support all the needed features for running a stable graph like e.g. flow limiter node. User can also run the graph in a server or run it inside application host. Here can be found more information about [MediaPipe framework ](https://developers.google.com/mediapipe/framework/framework_concepts/overview)
+
+This guide gives information about:
+
+* <a href="#how-to-build">How to build OVMS with MediaPipe support</a>
+* <a href="#ovms-calculators">OVMS Calculators</a>
+* <a href="#graph-proto">Graph proto files</a>
+* <a href="#configuration-files">Configuration files</a>
+* <a href="#using-mediapipe">Using the mediapipe graphs</a>
+* <a href="#graphs-examples">Graphs examples </a>
+* <a href="#current-limitations">Current Limitations</a>
+
+## How to build OVMS with mediapipe support <a name="how-to-build"></a>
+Building OVMS with mediapipe support requires passing additional flag for make command, for example:
+
+```
+MEDIAPIPE_DISABLE=0 make docker_build
+```
+
+More information about OVMS build parameters can be found here [here](build_from_source.md) 
+
+## Node Types <a name="ovms-calculators"></a>
+
+"Each calculator is a node of a graph. The bulk of graph execution happens inside its calculators. Ovms has its own calculators but can also use newly developed calculators or reuse the existing calculators defined in the oryginal mediapipe repository."
+
+For more details you can visit mediapipe concept description - [Calculators Concept Page](https://developers.google.com/mediapipe/framework/framework_concepts/calculators) or Ovms specific calculators implementation - [Ovms Calculators Concept Page](https://github.com/openvinotoolkit/model_server/blob/releases/2023/0/src/mediapipe_calculators/calculators.md)
+
+## Graph proto files <a name="graph-proto"></a>
+
+Graph proto files are used to define a graph. Example content of graph containing OVMSCalculator node proto file:
+
+```
+
+input_stream: "in"
+output_stream: "out"
+node {
+  calculator: "OVMSOVCalculator"
+  input_stream: "B:in"
+  output_stream: "A:out"
+  node_options: {
+        [type.googleapis.com / mediapipe.OVMSCalculatorOptions]: {
+          servable_name: "dummy"
+          servable_version: "1"
+          tag_to_input_tensor_names {
+            key: "B"
+            value: "b"
+          }
+          tag_to_output_tensor_names {
+            key: "A"
+            value: "a"
+          }
+        }
+  }
+}
```

Comment:
OVMSOVCalculator is mainly for our test purposes -> it uses directly C-API instead of ModelAPI Adapter approach.